docs/README.md

```diff
@@ -12,6 +12,6 @@ Flarum is the combined successor of [esoTalk](https://github.com/esotalk/esoTalk
 
 * **Beautiful and responsive.** This is forum software for humans. Flarum is carefully designed to be consistent and intuitive across platforms, out-of-the-box.
 
-* **Powerful and extensible.** Customize, extend, and integrate Flarum to suit your community. Flarum’s architecture is amazingly flexible, with a [powerful Extension API](/extend/).
+* **Powerful and extensible.** Customize, extend, and integrate Flarum to suit your community. Flarum’s architecture is amazingly flexible, with a [powerful Extension API](https://docs.flarum.org/extend/).
```

Comment:
I don't think this change should be here - the current link works fine and we shouldn't be hardcoding the URL.

Reply:
Yes, we shouldn't be hardcoding the URL. and it's better to keep using relative links.

Reply:
Yeah, it works on the site. Just not on the repo https://github.com/flarum/docs. But I can undo it.

Reply:
Because after the document is modified, if someone need a local online preview of the site, the relative link is required to open the latest local page.

Reply:
Let's undo it then 👍️

Reply:
I've undone the change. It was just a quick thing I've noticed and added to the PR along the way anyways.